tesseract_core/sdk/tesseract.py

```diff
@@ -440,7 +512,7 @@ def _request(
 
         return data
 
-    def _run_tesseract(self, endpoint: str, payload: dict | None = None) -> dict:
+    def run_tesseract(self, endpoint: str, payload: dict | None = None) -> dict:
```

Comment:
Why do we want this public as well? Just to use it in LocalClient?

Reply:
IMO it is a public method to this class (since it's called directly from other code). It's still not part of the public API, but that's a separate thing.

Reply:
Oops, this change was in fact accidental. Rolled it back

Reply:
it was not, but that's ok 👍